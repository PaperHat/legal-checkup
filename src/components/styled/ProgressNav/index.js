import styled from 'styled-components'

const ProgressNav = styled.nav`
  display: flex;
`

ProgressNav.displayName = 'StyledProgressNav'

export default ProgressNav
