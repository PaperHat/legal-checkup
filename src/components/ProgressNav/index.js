import React from 'react'

import { NavButton, ProgressBar } from '..'
import { StyledProgressNav } from '../styled'

export default function ProgressNav () {
  return (
    <StyledProgressNav>
      <NavButton type='back' />
      <ProgressBar />
      <NavButton type='forward' />
    </StyledProgressNav>
  )
}
