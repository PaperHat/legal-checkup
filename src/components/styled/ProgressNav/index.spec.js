import React from 'react'
import { shallow } from 'enzyme'

import ProgressNav from '.'

describe('components:styled:ProgressNav', () => {
  it('renders the ProgressNav and CSS properties properly', () => {
    expect(
      toJson(shallow(<ProgressNav>StyledProgressNav</ProgressNav>))
    ).toMatchSnapshot()
  })
})
