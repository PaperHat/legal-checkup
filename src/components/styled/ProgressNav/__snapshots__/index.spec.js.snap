// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components:styled:ProgressNav renders the ProgressNav and CSS properties properly 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

<nav
  className="c0"
>
  StyledProgressNav
</nav>
`;
